engine/utils/ocaml_of_json_schema/test/rust/src/lib.rs

```diff
@@ -0,0 +1,39 @@
+use schemars::JsonSchema;
```

Comment:
Please add some comments to all files in this crate what they are doing.

Reply:
I added a README that describes the thing, I think that's enough, what do you think?

Reply:
Generally all the code needs comments as well (readme is high level doc also for someone just using it, comments in the code is for anyone working on the code including future selves).
I'll complain about any undocumented code but won't block on it for now in favour of speed.

Reply:
Yes, I agree, I basically wrote no comment while rushing adding features.
Ok, nice, thanks!